content/en/getting-started/help-and-support/index.md

```diff
@@ -10,89 +10,120 @@ cascade:
 
-LocalStack Slack Community includes LocalStack users, contributors, and maintainers.
-If you need help with the community version of LocalStack, please use the `#help` channel.
-You can sign up for the [LocalStack Slack Community](https://localstack.cloud/slack) by creating an account.
+## Support plans
 
-However, the messages on Slack are not accessible after three months, so it is not the best place to ask questions that may be useful to others in the future.
-For that, we recommend using LocalStack Discuss.
+Find the details about different support services and what they entitle bellow.
+Please refer to the [Support Features](#support-features) to see which features are included in each plan.
 
-### GitHub Issue
+### Basic support
 
-You can use GitHub Issue to:
+Basic Support is included for all LocalStack users, and provides access to our documentation, community support, account management support (operational support).
 
-- [Request new features](https://github.com/localstack/localstack/issues/new?assignees=&labels=type%3A+feature%2Cstatus%3A+triage+needed&template=feature-request.yml&title=feature+request%3A+%3Ctitle%3E)
-- [Report existing bugs](https://github.com/localstack/localstack/issues/new?assignees=&labels=type%3A+bug%2Cstatus%3A+triage+needed&template=bug-report.yml&title=bug%3A+%3Ctitle%3E)
+You can avail community support through the following channels:
 
-Make sure to follow the issue templates and provide as much information as possible.
-If you have encountered outdated documentation, please report it on our [documentation GitHub page](https://github.com/localstack/docs).
+- [LocalStack Discuss](https://discuss.localstack.cloud/)
+- [LocalStack Slack Community](https://localstack.cloud/slack)
+- [GitHub Issue](https://github.com/localstack/docs/issues/new)
 
-## Dedicated support
+Community support is provided on a best-effort basis and is not guaranteed.
+We also encourage you to help others in the community by answering questions and sharing your experiences.
 
-Dedicated customer support is available to **Starter** & **Teams** plan users with an active subscription.
+### Standard support
 
-The support team is available to help you with:
+Standard support is available to all LocalStack users with an active Starter or Trial subscription.
+With standard support, you get access to 1v1 support based on best effort basis.
+With the best effort support, we do not provide fixed response times.
+However, as we understand the importance of timely assistance, our support team always strives to respond to inquiries and requests within 24–48 hours during our [regular business hours](#support-business-hours).
 
-- Troubleshooting LocalStack issues
-- Answering questions about LocalStack features and functionality
-- Providing guidance on how to use LocalStack
-- Providing guidance on how to integrate LocalStack with your application
+#### Troubleshooting Guidelines
```

Comment:
This whole section needs to be moved under the mentioned sections. Right now it's cutting through the `Support plans` section.

```
### Priority Support
### Enterprise Support
```